### ISSUE TITLE
SingleChronicleQueue#entryCount() fails with IllegalArgumentException

### ISSUE THREAD

**Issue by abysas** [OpenHFT/Chronicle-Queue#579]
`SingleChronicleQueue#entryCount()` fails:
```
java.lang.IllegalArgumentException: Unsigned Int 31-bit -2147483648 out of range

	at net.openhft.chronicle.core.Maths.toUInt31(Maths.java:367)
	at net.openhft.chronicle.queue.RollCycles.toCycle(RollCycles.java:100)
	at net.openhft.chronicle.queue.impl.single.SingleChronicleQueue.countExcerpts(SingleChronicleQueue.java:522)
	at net.openhft.chronicle.queue.impl.single.SingleChronicleQueue.entryCount(SingleChronicleQueue.java:634)
	at ...
```
The test case:
```
    @Test
    void testEntryCount() {
        SingleChronicleQueue queue = ChronicleQueue.singleBuilder(new File(System.getProperty("user.dir"))).build();
        queue.entryCount();
    }
```
Chronicle Queue version 5.17.3

**Comment by abysas** [comment]
Please comment, why this issue was closed?

**Comment by dpisklov** [comment]
@abysas because it's fixed may be?

**Comment by abysas** [comment]
@dpisklov, sorry, didn't noticed that ticket was closed as a result of the commit. Thanks a lot for the fix.

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.9](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.9), [BOM-2.17.149](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.149)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.9](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.9), [BOM-2.17.196](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.196)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.9](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.9), [BOM-2.17.225](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.225)

**Comment by hft-team-city** [comment]
Released in [Chronicle-Queue-5.17.9](https://github.com/OpenHFT/Chronicle-Queue/releases/chronicle-queue-5.17.9), [BOM-2.17.241](https://github.com/OpenHFT/OpenHFT/releases/chronicle-bom-2.17.241)
